### PR TITLE
Fixed JuLabat resize bug and perfs

### PR DESCRIPTION
**Pull request by julienlabat**
3h du mat' sa mère … 🦉

### CODE REVIEW

File: src/styles/julabat.js [hunk 1]
```diff
@@ -5,94 +5,99 @@
...
     // Create star with random chance depending on noise value
-    const f = this.noiseField.reduce((a, v) => {
-      const r = FXRandomBetween(0, 1)
-      if (v.z < r * 0.5 && v.z < avg) a.push(v)
+    const f = noiseField.reduce((a, v) => {
+      if (v.z < P5.random() * 0.5 && v.z <= avg) a.push(v)
       return a
     }, [])
 
+    console.log(f.length)
```
Comment (camilleroux) [inline src/styles/julabat.js:67]: Can you remove all the useless console.log call please?

File: src/styles/julabat.js [hunk 2]
```diff
@@ -123,28 +128,37 @@ export default class JuLabatStyle extends Style {
...
     // Reset effects
-    this.setBlur(0)
+    this.removeBlur()
     P5.blendMode(P5.BLEND)
   }
 
-  afterDraw () {}
+  afterDraw () {
+    const P5 = this._p5
+    if (P5.frameCount > 1) P5.noLoop()
```
Comment (camilleroux) [inline src/styles/julabat.js:151]: noLoop() is already called in index.js.

File: src/styles/julabat.js [hunk 3]
```diff
@@ -123,28 +128,37 @@ export default class JuLabatStyle extends Style {
...
+  }
 
   setBlur (size) {
     this._p5.drawingContext.filter = `blur(${size}px)`
   }
 
+  removeBlur () {
+    this._p5.drawingContext.filter = 'none'
```
Comment (camilleroux) [inline src/styles/julabat.js:159]: I added this in index.js, it shouldn't be necessary

**Review by camilleroux — CHANGES_REQUESTED** [review]

File: src/styles/julabat.js (hunk 1 above)
Comment (julienlabat) [reply src/styles/julabat.js:67]: I missed this one, removing it right now

File: src/styles/julabat.js (hunk 2 above)
Comment (julienlabat) [reply src/styles/julabat.js:151]: It's the only trick I found to resolve my resizing issue. I didn't found out exactly what caused the problem but I solved it by enabling the script to loop for just one frame.

Comment (julienlabat) [reply src/styles/julabat.js]: Ok everything is fixed now thanks to @solarisedesign

**Pull request merged**